.github/CONTRIBUTING.md

```diff
@@ -290,3 +305,14 @@ Git can append the `Signed-off-by` line to your commit messages. Simply
 provide the `-s` or `--signoff` option when performing a `git commit`.
 For more information about writing commit messages, visit [How to Write
 a Git Commit Message](https://chris.beams.io/posts/git-commit/).
+
+#### Co-authored By
+If someone else had part in your pull request, please add the following to the commit:
+`Co-authored-by: Name <gitregistered@email.adress>`
```

Comment:
adress > address

Reply:
Sometimes, after reading it 6 times, I still question myself if I actually read it at all :P